server/streamable_http.go

```
@@ -0,0 +1,810 @@
+
+	// Set SSE headers
+	w.Header().Set("Content-Type", "text/event-stream")
+	w.Header().Set("Cache-Control", "no-cache")
+	w.Header().Set("Connection", "keep-alive")
+	w.WriteHeader(http.StatusOK)
+
+	// Generate a unique ID for this stream
+	s.standaloneStreamID = uuid.New().String()
+
+	// Send an initial event to confirm the connection is established
+	initialEvent := fmt.Sprintf("data: {\"jsonrpc\": \"2.0\", \"method\": \"connection/established\"}\n\n")
```

golangci-lint (1.64.8)

S1039: unnecessary use of fmt.Sprintf

(gosimple)